rmgpy/molecule/group.py

```diff
@@ -1909,7 +2008,27 @@ def find_subgraph_isomorphisms(self, other, initial_map=None, save_order=False):
         else:
             if group.multiplicity:
                 return []
-
+        if self.metal:
+            for mult1 in self.metal:
```

Comment:
Same comment about choice of variable names (and cython type declarations)

Reply:
Defined m1,m2 as strings and used them for metal and facet double loops.